csrc/dd_arith.h

```diff
@@ -506,11 +508,34 @@ static inline ddouble hypotqd(ddouble x, double y)
     return hypotqq(x, (ddouble){y, 0});
 }
 
+/* Computes the nearest integer to d. */
+inline ddouble nintq(ddouble d) {
```

Comment:
This needs to be static inline, because of C weird inline rules
```suggestion
static inline ddouble nintq(ddouble d) {
```

Reply:
Static has been added.